pod.go

```diff
@@ -152,6 +155,18 @@ func (pc *podCollector) Describe(ch chan<- *prometheus.Desc) {
 	ch <- descPodContainerResourceLimitsMemoryBytes
 }
 
+func printControllers(annotation map[string]string) string {
```

Comment:
Though CreatedByAnnotation represents the key used to store the spec(json)
used to create the [resource](https://github.com/kubernetes/client-go/blob/65ac4e638a79d49b7dbd5e1cef4e7785af7cbd39/pkg/api/helpers.go#L440) ,controllers named come from `kubeclt describe po xxx`